packages/launchpad/src/setup/OpenBrowser.vue

```diff
@@ -106,6 +119,19 @@ const launch = async () => {
   }
 }
 
+const launchIfBrowserSetInCli = async () => {
+  await lsQuery
+  const isValidBrowser = lsQuery.data.value?.localSettings.preferences.isValidBrowser
+  const wasBrowserSetInCli = lsQuery.data.value?.localSettings.preferences.wasBrowserSetInCLI
+  const globalLaunchCount = lsQuery.data.value?.localSettings.preferences.globalLaunchCount
+
+  if (wasBrowserSetInCli && isValidBrowser && (globalLaunchCount === 0)) {
```

Comment:
Rather than exposing these new fields and having this business logic in the front end, can we instead expose a dedicated field like `shouldLaunchBrowserFromOpenBrowser` and have this logic abstracted in the server?

Reply:
@ryanthemanuel now that you point that out, that seems more reasonable to do. Thank you! 

As far as creating a new field...
Would the best place for this field be in `localSettings` that is already defined for graphQL or should a create a new object type and standalone file for the field?